### DISCUSSION TITLE
画像クリックorタップ時の挙動改善について

### DISCUSSION THREAD

**Discussion by ichii731** [increments/qiita-discussions#78] [要望・アイデア・不具合]
よくQiitaを利用させていただいております。
Qiitaの改善案の一例として画像をタップorクリックした際の挙動の変更を提案させて頂きます。
現状、ページ内の画像をクリックしたらqiitausercontentの画像アドレスへと直接飛ばされる仕様です。
![Screenshot_20211206_132450](https://user-images.githubusercontent.com/59349039/144787044-a87b419d-11fa-4aaa-8f8f-5841169ec9c5.jpg)
もしよろしければLightboxなどを使ったページ内で画像を拡大・表示出来るような機能を実装してほしいです。
ご検討いただけないでしょうか。

**Comment by kskwtnk** [comment]
@ichii731 

ご意見ありがとうございます！

ちなみに、表示が変更されると嬉しい理由やモチベーションについてもう少し伺ってもよろしいでしょうか？
実装的にはもちろん可能だと思うのですが、今まであまりいただいたことがないカテゴリーのご意見でしたのでもう少し詳細に教えていただけたら幸いです！

**Reply by ichii731** [reply]
ご返信有難うございます。

新規タブで画像を開く形になるため、どうしても画像をより詳しく見たい際にワンステップ別の操作が必要となり、加えて近年殆どのページで画像のポップアップが使われていることから単純に大きく表示されるようにできればなぁ…と思った次第です。

また、アドカレ含めググる時以外は基本的にQiitaはPWAを通じて見るのですが、Android版**Chromeでは画像URLにアクセスした際にダブルタップで拡大出来ないという仕様**になってまして、ピンチ操作が必要になり閲覧しづらくなると感じました。（Operaなどはダブルタップで拡大できます）
※参考動画：https://www.youtube.com/watch?v=o4zEqQuFAzY

他の利用者さんにも似たようなご意見などありましたら、何時でも構いませんので是非ご検討いただきたいという提案でした。
実装することでページ自体の容量が多くなるのでしたら、PWAからのアクセスした場合や設定でONにした場合にのみ有効にさせる…的な対応でも良いのではないかと思っております。

**Reply by kskwtnk** [reply]
@ichii731 

詳細にありがとうございます！
結論としては「実装はできるものの、一旦他の方の意見も募りたいのでお待ちください」となります、すみません！ 🙏 

私が個人的にあまり画像を拡大して見ないのもあり認識が弱かったですが、たしかに仰るとおりですね
ただ一方でLightboxのような形式ですと記事を見るには画像を閉じる必要があるので「画像と記事、両方を開いて見たい」という方もいるのかもしれないといった議論になりました

こちらの件について、他の方からも是非ご意見いただきたいです！
モバイルのときはこうが良いけどデスクトップではこうが良い……みたいなものでも歓迎です！

（実装自体はできるので）運営としては要望の量を確認したいなと思っています 🙇

**Reply by ichii731** [reply]
転送量も影響するので直ぐに！という形では無く色々な意見も踏まえて対応の可否を判断して頂ければ幸いです。今後とも宜しくお願いします！
（あとQiita株式会社に名前変えてたんですね…）

**Reply by kskwtnk** [reply]
@ichii731 

> 転送量も影響するので直ぐに！という形では無く色々な意見も踏まえて対応の可否を判断して頂ければ幸いです。

色々と鑑みていただけて幸いです、ありがとうございます！

そして、12/1より社名を変更いたしました！
今後ともよろしくお願いします！
